appcues/src/main/java/com/appcues/statemachine/Error.kt

```diff
@@ -1,12 +1,15 @@
 package com.appcues.statemachine
 
 import com.appcues.data.model.Experience
+import com.appcues.data.model.RenderContext
 import java.util.UUID
 
 internal sealed class Error(open val message: String) {
+
     val id: UUID = UUID.randomUUID()
 
-    data class ExperienceAlreadyActive(val ignoredExperience: Experience, override val message: String) : Error(message)
+    data class ExperienceAlreadyActive(val experience: Experience) : Error("Experience ${experience.name} already active")
+    data class RenderContextNotActive(val renderContext: RenderContext) : Error("RenderContext $renderContext is not active.")
```

Comment:
small quality of life change here, since both errors ExperienceAlreadyActive and RenderContextNotActive only have a single Message, we can abstract it and have a hardcoded value here

Reply:
I think a subtle bug here maybe - the parameter for ExperienceAlreadyActive is the ignored experience - `ignoredExperience` and the message is now saying that experience is already active, rather than the name of the experience that is active? I guess it should just go back to the previous value
```
"Experience already active"
```
from L135 of StateMachine previous version, and keep the name `ignoredExperience` to make that clear

Reply:
it makes sense, since we are eventually logging this one I wish we could use the ignoredExperience value as part of the message, or else we could just remove the val ignoredExperience from the class since its not used anywhere.

I was maybe thinking of"

"Experience already active, experience ..... ignored."